src/Sarif/NotYetAutogenerated/SuppressionStates.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft.  All Rights Reserved.
+// Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.CodeDom.Compiler;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    /// <summary>
+    /// The state of a result relative to a baseline of a previous run.
+    /// </summary>
+    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "0.16.0.0")]
+    [Flags]
+    public enum SuppressionStates
+    {
+        None,
```

Comment:
I agree that it's typical. I'm saying two things:

1. In this particular case, "None" didn't convey _to me_ the meaning "no information about suppression". To me, it conveyed "no suppression".

2. The guidelines allow you wiggle room on this point:

> **DO** provide a value of zero on simple enums.
>
> _Consider_ calling the value _something like_ "None" [my emphasis]

Given that, please consider renaming to Unknown, but I'll accept your decision.